openassessment/xblock/submission_mixin.py

```diff
@@ -113,9 +115,20 @@ def submit(self, data, suffix=''):  # pylint: disable=unused-argument
 
         status_tag = 'ENOMULTI'  # It is an error to submit multiple times for the same item
         status_text = self._(u'Multiple submissions are not allowed.')
+
         if not workflow:
             try:
-                submission = self.create_submission(student_item_dict, student_sub_data)
+
+                # a submission for a team generates matching submissions for all members
+                if self.teams_enabled:
```

Comment:
I think you might actually want `self.is_team_assignment()` here (which also checks `self.team_submissions_enabled`).